lms/djangoapps/student_account/views.py

```diff
@@ -67,16 +62,6 @@ def login_and_registration_form(request, initial_mode="login"):
     # Retrieve the form descriptions from the user API
     form_descriptions = _get_form_descriptions(request)
 
-    # If this is a microsite, revert to the old login/registration pages.
-    # We need to do this for now to support existing themes.
-    # Microsites can use the new logistration page by setting
-    # 'ENABLE_COMBINED_LOGIN_REGISTRATION' in their microsites configuration file.
-    if microsite.is_request_in_microsite() and not microsite.get_value('ENABLE_COMBINED_LOGIN_REGISTRATION', False):
-        if initial_mode == "login":
-            return old_login_view(request)
-        elif initial_mode == "register":
-            return old_register_view(request)
```

Comment:
@waheedahmed removing this check will work, but there are other parts of the codebase that check the value of `ENABLE_COMBINED_LOGIN_REGISTRATION`, too. It may also be worth checking with @douglashall about deployment strategies. Leaving this check here will allow us to more easily roll back to the old pages if something goes wrong.